### ISSUE TITLE
ALOpsAppCompiler : Problem with "LoadFrom"

### ISSUE THREAD

**Issue by ldumont13** [HodorNV/ALOps#121]
Hi guys ! 

We are trying to create / build your first pipeline without using Docker but in the step of compilation we got this error " Exception when calling "LoadFrom" with "1" argument (s): "Invalid directory on the URL. "
![image](https://user-images.githubusercontent.com/44373612/80507634-58136100-8977-11ea-8d77-46dee302d130.png)

Can you help us, if you see something wrong or if something is missing on your YAML :
![image](https://user-images.githubusercontent.com/44373612/80508135-f0a9e100-8977-11ea-9358-7ab512ff20f2.png)
We tried many things and option for each step but we got always the same error. But the step Import Licence  is successful.

Best Regards,
Lucas D.

**Comment by AdminHodor** [comment]
Dear @ldumont13 ,

After investigating we found the probably cause of the issue. The Tasks does try to import some DLL's from the ServiceTier folder, very likely your NST Service is located on a different path than the default.

To mitigate we made a hotfix for this module so the "NST Folder" will be resolved automatically based on the "ServerInstance" name. This way the task should work without any dependency on paths.

We should be able to get the hotfix out this week, we'll keep you updated via this issue.

Kind regards,
